utils/swanmon/Makefile

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright (C) 2023 TDT AG <development@tdt.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See https://www.gnu.org/licenses/gpl-2.0.txt for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=swanmon
+PKG_VERSION:=0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/TDT-AG/swanmon
```

Comment:
Will there be some documentation how to use this package or readme in the repository?

Reply:
I have now added more documentation including examples in the repository.